renormalizer/mps/solver.py

```diff
@@ -7,7 +7,7 @@
 import scipy
 
 
-from renormalizer.lib.davidson import davidson
+from renormalizer.lib import davidson
```

Comment:
why is this import different from the style in /lib/__init__.py 
"from renormalizer.lib.davidson.davidson import davidson"?

Reply:
In `lib/init.py` it can be understood as `from .davidson.davidson import davidson`. 
After `lib/init.py` is executed, the `davidson` function appears under `renormalizer.lib` and we can import this way here.

Reply:
got it.